FiducialRegistrationWizard/Logic/vtkSlicerFiducialRegistrationWizardLogic.h

```diff
@@ -84,9 +84,16 @@ class VTK_SLICER_FIDUCIALREGISTRATIONWIZARD_MODULE_LOGIC_EXPORT vtkSlicerFiducia
   virtual void OnMRMLSceneNodeRemoved(vtkMRMLNode* node);
 
 private:
+
   vtkSlicerFiducialRegistrationWizardLogic(const vtkSlicerFiducialRegistrationWizardLogic&); // Not implemented
   void operator=(const vtkSlicerFiducialRegistrationWizardLogic&);               // Not implemented
 
+  static double SumOfSquaredElementsInArray( vtkDoubleArray* array );
```

Comment:
For me, it's not obvious how all this works. Adding a one-sentence description for each method would help a lot.